internal/operators/ocs/utils.go

```diff
@@ -0,0 +1,3585 @@
+package ocs
+
+import (
+	"bytes"
+	"strconv"
+	"text/template"
+)
+
+func ocsSubscription(OpenShiftVersion string) (string, error) {
+	data := map[string]string{
+		"OPENSHIFT_VERSION": "stable-" + OpenShiftVersion,
```

Comment:
Why do we need the "stable"? You did not add it in LSO patch.

Reply:
@rollandf  
The channel for LSO is (4.5,4.6 ... and so), while for OCS it is (stable-4.6, stable-4.6 ..) . For ref: 
```
NAMESPACE                 NAME                     PACKAGE                  SOURCE              CHANNEL
openshift-local-storage   local-storage-operator   local-storage-operator   redhat-operators    4.6
openshift-storage         ocs-subscription         ocs-operator             ocs-catalogsource   stable-4.6
```

Reply:
Weird, why is it different?
Is this similar if you install from UI?

Reply:
yes! this is similar from the UI too!

Reply:
Thanks for clarfiying